OpenRA.Game/Game.cs

```diff
@@ -232,6 +232,21 @@ public static void RestartGame()
 			// Reseed the RNG so this isn't an exact repeat of the last game
 			lobbyInfo.GlobalSettings.RandomSeed = CosmeticRandom.Next();
 
+			// Update MapCache
```

Comment:
Can we please change this to say something like
```
// Note: the map may have been changed on disk outside the game, changing its UID.
// Use the updated UID if we have tracked the update instead of failing.
```
?

Reply:
sure